docs/modules_packages_crates/crates_and_packages.md

```diff
@@ -28,6 +31,8 @@ A Nargo _package_ is a collection of one of more crates that provides a set of f
 
 A package _must_ contain either a library or a binary crate, but not both.
 
+Binary packages (but not library packages) can be defined as a [workspace](./workspaces).
+
```

Comment:
Libraries can be defined in a workspace. We just don't have a way to consume libraries from inside a workspace as external dependencies right now.

Inside a workspace, these are consumed as `{ path = ".../" }` dependencies